src/appleseed.studio/mainwindow/renderingsettingswindow.cpp

```diff
@@ -834,6 +834,7 @@ namespace
 
             if (!filepath.isEmpty())
             {
+                filepath = QDir::toNativeSeparators(filepath);
```

Comment:
I don't think the change should be done here. I think it's better to do this in `get_open_filename`

Reply:
That's a good point. This should be moved to `get_open_filename()` and removed from all call sites to `get_open_filename()`. This can be done in a separate PR.

Reply:
-> https://github.com/appleseedhq/appleseed/issues/2778